charts/README.md

```diff
@@ -5,15 +5,24 @@
 
 ## Prerequisites
 
-- Kubernetes 1.4+ with Beta APIs enabled
+- Kubernetes 1.6+ with Beta APIs and RBAC enabled
```

Comment:
Do we need RBAC actually *enabled*?  It's enough for the cluster to be 1.6+, right?